appveyor.yml

```diff
@@ -53,6 +53,7 @@ install:
     # Add path, activate `conda` and update conda.
     - cmd: call %CONDA_INSTALL_LOCN%\Scripts\activate.bat
     - cmd: conda update --yes --quiet conda
+    - cmd: conda clean --all
```

Comment:
Please do not directly modify any files outside of the `recipe` directory. If you are concerned about an issue related to them, please try [re-rendering]( https://github.com/conda-forge/conda-smithy#re-rendering-an-existing-feedstock ). If that doesn't work, please ping core. Thanks.

Reply:
OK sure. Thanks for the tag. I will undo this change now.

Reply:
I think the re-rendering fixed it. So shouldn't need further adjustment.

Reply:
thanks for the help @jakirkham.